dht/token.hh

```diff
@@ -65,12 +68,20 @@ public:
     //     [0x80] == 0.5
     //     [0x00, 0x80] == 1/512
     //     [0xff, 0x80] == 1 - 1/512
-    managed_bytes _data;
+    std::array<uint8_t, 8> _data;
```

Comment:
Why not `int64`?

Reply:
It's an intermediate step to split the change. The following patch turns it into int64_t.